docs/CHARTER.md

```diff
@@ -0,0 +1,113 @@
+
+ii. Committers are Contributors who have earned the ability to modify (“commit”) source code, documentation or other technical artifacts in a project’s repository; and
+
+iii. A Contributor may become a Committer by a majority approval of the existing Committers. A Committer may be removed by a majority approval of the other existing Committers.
+
+d. Participation in the Project through becoming a Contributor and Committer is open to anyone so long as they abide by the terms of this Charter. 
+
+e. The TSC may (1) establish work flow procedures for the submission, approval, and closure/archiving of projects, (2) set requirements for the promotion of Contributors to Committer status, as applicable, and (3) amend, adjust, refine and/or eliminate the roles of Contributors, and Committers, and create new roles, and publicly document any TSC roles, as it sees fit.
+
+f. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and will serve until their resignation or replacement by the TSC.** ** The TSC Chair, or any other TSC member so designated by the TSC, will serve as the primary communication contact between the Project and OpenSSF, a directed fund of The Linux Foundation.
+
+g. Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
+
+i. coordinating the technical direction of the Project;
+
+ii. approving project or system proposals (including, but not limited to, incubation, deprecation, and changes to a sub-project’s scope);
+
+iii. organizing sub-projects and removing sub-projects;
+
+iv. creating sub-committees or working groups to focus on cross-project technical issues and requirements;
+
+v. appointing representatives to work with other open source or open standards communities;
+
+vi. establishing community norms, workflows, issuing releases, and security issue reporting policies;
+
+vii. approving and implementing policies and processes for contributing (to be published in the CONTRIBUTING file) and coordinating with the series manager of the Project (as provided for in the Series Agreement, the “Series Manager”) to resolve matters or concerns that may arise as set forth in Section 7 of this Charter;
+
+viii. discussions, seeking consensus, and where necessary, voting on technical matters relating to the code base that affect multiple projects; and
+
+ix. coordinating any marketing, events, or communications regarding the Project.
+
+## 3. TSC Voting
+
+a. While the Project aims to operate as a consensus-based community, if any TSC decision requires a vote to move the Project forward, the voting members of the TSC will vote on a one vote per voting member basis.
+
+b. Quorum for TSC meetings requires at least fifty percent of all voting members of the TSC to be present. The TSC may continue to meet if quorum is not met but will be prevented from making any decisions at the meeting.
```

Comment:
Concerns raised over the TSC being able to meet quorum on a consistent basis. A couple of options were discussed. @fkautz recommended a common compromise of _members are temporarily dropped from the quorum requirement based upon attendance._   

Typical language I have seen used in several other communities for this model is:

- If a TSC member misses any two consecutive TSC meetings, such member shall not be counted against the quorum for voting purposes until that member next attends a TSC meeting. 

-kenny

Reply:
discussed Apr. 10 - decision to use the suggested language and rely on electronic voting model when necessary.

Reply:
I pushed a commit adding the language discussed during the meetting.